index.html

```diff
@@ -3197,14 +3197,36 @@ <h3>Reserved Extension Points</h3>
 see Section <a href="#types"></a>.
         </p>
 
+        <p class="issue" title="(AT RISK) Reserved properties">
+The following reserved properties are at risk and will be removed from the
+specification if at least one specification with two independent implementations
+are not demonstrated by the end of the Candidate Recommendation Phase:
+`confirmationMethod` / `confidenceMethod`, `presentationSchema`, and
+`renderMethod`.
+        </p>
```

Comment:
It isn't clear to me if this means only from the pertinent sections of the spec, or from this table as well.
Would it be better to put this text before each  at risk property?

Reply:
> It isn't clear to me if this means only from the pertinent sections of the spec, or from this table as well. 

Yeah, good point, I wasn't thinking about the existing sections of the spec.

We should probably just add at-risk markers to `evidence`, `refreshService`, and `termsOfUse` to be crystal clear wrt. their current state (at risk) as we enter CR. I'll do that in a separate PR.

> Would it be better to put this text before each at risk property?

Yes, I can do that. It felt like it would be repetitive to copy/paste the same text to each entry. In the grand scheme of things, it doesn't really matter because the entry and/or text would be removed as we move into PR. I'll do the revision in an update to this PR.

Reply:
Fixed in https://github.com/w3c/vc-data-model/pull/1108/commits/a86f02053c1063164671a57d7962b868a916b90b.